themes/openy_themes/openy_lily/scripts/vendor/jquery.matchHeight/jquery.matchHeight-min.js

```diff
@@ -1,12 +0,0 @@
-/*
```

Comment:
why it was deleted? We are using it for mobile view of ALerts
Ensure ```openy_system/jquery.matchHeight``` added as dependency to alerts library, please